pyproject.toml

```diff
@@ -18,3 +18,10 @@ line_length = 79
 namespace_packages = false
 python_version = "3.8"
 strict = true
+
+[tool.codespell]
+# Ref: https://github.com/codespell-project/codespell#using-a-config-file
+skip = '.git*'
```

Comment:
There's some stuff in `.github` that might be worth checking.  Is this just for `.gitattributes`?

Reply:
frankly there was some issue with codespell not skipping `.git` folder.  I will make it more explicit and point only to .git.